code/modules/SCP/SCPs/SCP-173.dm

```diff
@@ -272,7 +272,7 @@ GLOBAL_LIST_EMPTY(scp173s)
 	if(feces_amount >= 60) // Breach, gonna take ~30 minutes
```

Comment:
Update this comment to reflect the shit timer change, please

Reply:
Done.